php/checkAnswer.php

```diff
@@ -0,0 +1,25 @@
+<?php
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+require_once('connection.php');
+
+$quizID = $_POST['quizID'];
+$questionID = $_POST['questionID'];
+$questionNumber = $_POST['questionNumber'];
+$userChoice = $_POST['userChoice'];
+
+$conn = newConn();
+
+// Get the correct answer from the database
+$stmt = $conn->prepare("SELECT correctAnswer FROM questions WHERE questionID = :questionID");
```

Comment:
Make a prepared statement for this.

Reply:
Could you clarify this please, I think it's already in a prepared statement but correct me if wrong

Reply:
I meant a prepared procedure/function.
Your SQL should look like "CALL myFunction();" and not what you have now.

Reply:
Ah, got it. Will do that now